Add guide on how to remove clusters stuck in pending state. (#3309)
mesosphere/dcos-docs-site: pages/ksphere/kommander/1.2/troubleshooting/index.md

> Similarly, to diagnose other missing actions for each resource listed above, the corresponding `update`, `delete`, and `create` permissions can be substituted in the place of `list` in the provided commands.

Reviewer: ```suggestion
Use the `update`, `delete`, and `create` permissions, like the `list` command is used, to diagnose other missing actions for each resource.
```

Reviewer: Please use the same comments I gave for 1.1, for 1.2.

> ### I cannot detach an attached cluster that is "Pending"

Reviewer: ```suggestion
## I cannot detach an attached cluster that is "Pending"
```
